Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -1793,6 +1793,60 @@ class PropertyWrapperColor : public AnimationPropertyWrapperBase {
     void (RenderStyle::*m_setter)(const Color&);
 };
 
+class ScrollbarColorPropertyWrapper final : public AnimationPropertyWrapperBase {
+    WTF_MAKE_FAST_ALLOCATED;
+public:
+    ScrollbarColorPropertyWrapper()
+        : AnimationPropertyWrapperBase(CSSPropertyScrollbarColor)
+        , m_thumbWrapper(makeUnique<PropertyWrapperStyleColor>(CSSPropertyScrollbarColor, &RenderStyle::scrollbarThumbColor, &RenderStyle::setScrollbarThumbColor))
+        , m_trackWrapper(makeUnique<PropertyWrapperStyleColor>(CSSPropertyScrollbarColor, &RenderStyle::scrollbarTrackColor, &RenderStyle::setScrollbarTrackColor))
+    {
+    }
+
+protected:
```

Comment:
A `final` class does not need anything `protected`. Should just be `private`. Not important though.